app/models/subnet_domain.rb

```diff
@@ -2,9 +2,6 @@ class SubnetDomain < ActiveRecord::Base
   belongs_to :domain
   belongs_to :subnet
 
-  validates :subnet_id, :presence => true
-  validates :domain_id, :presence => true, :uniqueness => {:scope => :subnet_id}
```

Comment:
how do you validate it then? IMHO this needs to stay and error should propagate to the calling object if its not already?